lib/quality/rake/task.rb

```diff
@@ -80,7 +80,7 @@ def ratchet_name
 
       def define
         desc 'Verify quality has increased or stayed ' \
-             'the same' unless ::Rake.application.last_comment
+             'the same' unless ::Rake.application.last_description
```

Comment:
And hound-ci doesn't seem to be respecting the rubocop config, anyway, so I'd ignore it regardless!